scripts/firedrake-install

```diff
@@ -59,7 +59,7 @@ class FiredrakeConfiguration(dict):
 
 def deep_update(this, that):
     import collections
-    for k, v in that.iteritems():
+    for k, v in that.items():
```

Comment:
Oh, you can't use `from six import iteritems` because it may not be installed yet?